pkg/multiTenancyPlugins/authorization/volumes.go

```diff
@@ -40,21 +38,21 @@ func filterVolumes(r *http.Request, rec *httptest.ResponseRecorder) []byte {
 	}
 	return buf.Bytes()
 }
-func hostFSMountCheck(tenantId string, volumeBindings[] string) (error) {
-	for _,b := range volumeBindings {
-		if index := strings.Index(b,":"); index > -1 {
-		  v := b[0:index]
-		  if strings.Contains(v,"/") {
-			return errors.New(HOST_FS_MOUNT_NOT_ALLOWED)
-		  }
+func hostFSMountCheck(tenantId string, volumeBindings []string) error {
+	for _, b := range volumeBindings {
+		if index := strings.Index(b, ":"); index > -1 {
```

Comment:
> From: Doron Podoleanu <notifications@github.com>
> To: swarm-hooks/swarm <swarm@noreply.github.com>
> Cc: Kenneth Nagin/Haifa/IBM@IBMIL, Author <author@noreply.github.com>
> Date: 20/07/2016 10:45 AM
> Subject: Re: [swarm-hooks/swarm] Volume binding (#46)
> 
> In pkg/multiTenancyPlugins/authorization/volumes.go:
> > @@ -40,21 +38,21 @@ func filterVolumes(r *http.Request, rec 
> *httptest.ResponseRecorder) []byte {
> >     }
> >     return buf.Bytes()
> >  }
> > -func hostFSMountCheck(tenantId string, volumeBindings[] string) 
(error) {
> > -   for _,b := range volumeBindings {
> > -      if index := strings.Index(b,":"); index > -1 {
> > -        v := b[0:index]
> > -        if strings.Contains(v,"/") {
> > -         return errors.New(HOST_FS_MOUNT_NOT_ALLOWED)
> > -        }
> > +func hostFSMountCheck(tenantId string, volumeBindings []string) error 
{
> > +   for _, b := range volumeBindings {
> > +      if index := strings.Index(b, ":"); index > -1 {

> See comment on the other use of this method - IMHO use regexp if 
possible
Personally,  I feel the usage on regexp in this case would be more 
confusing.  Ezra what is your preference?
--Ken